FindVitis.cmake

```diff
@@ -209,7 +209,6 @@ endforeach()
 
 FIND_PACKAGE_HANDLE_STANDARD_ARGS(Vitis HANDLE_COMPONENTS REQUIRED_VARS
 		VITIS_ROOT
```

Comment:
this is a long story related to the docker image for vitis in CI in mlir-aie and I don't need it merged anymore but I do want to clarify: that's not this change does; quoting the docs

```
This command handles the REQUIRED, QUIET and version-related arguments of [find_package()](https://cmake.org/cmake/help/latest/command/find_package.html#command:find_package). It also sets the <PackageName>_FOUND variable. The package is considered found if all variables listed contain valid results, e.g. valid filepaths.
```

So the only thing this changes is `VITIS_VPP_FOUND` and _whether `VITIS_VPP` is required_, which is what I needed to avoid in the CI docker image (which has all of chess and etc but not `v++`).

Reply:
I think the right thing is to decompose this, so that rather than looking for Vitis, we look for the individual components (e.g. chess, bootgen, xclbinutil)

Reply:
Possibly but I think in the meanwhile (at least as far as the docker image is concerned) @fifield has found a better solution: https://github.com/Xilinx/mlir-aie/pull/812/commits/02acd9bd1c8454c6685d0ee901dd7c96b53778df